inspector/src/components/actionTabs/tabs/propertyGrids/animations/animationCurveEditorComponent.tsx

```diff
@@ -1,28 +1,420 @@
 import * as React from "react";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faTimes } from "@fortawesome/free-solid-svg-icons";
+import { Animation } from 'babylonjs/Animations/animation';
+import { EasingFunction, BezierCurveEase } from 'babylonjs';
+import { AnchorPoint } from './anchorPoint';
+import { KeyframePoint } from './keyframePoint';
+
+require("./curveEditor.scss");
 
 interface IAnimationCurveEditorComponentProps {
     close: (event: any) => void;
     title: string;
+    animations: Animation[];
+    entityName: string;
+}
+
+interface KeyFrame {
+    frame: number;
+    value: number;
+    inTangent?: BABYLON.Vector2 | BABYLON.Vector3 | BABYLON.Quaternion;
+    outTangent?: BABYLON.Vector2 | BABYLON.Vector3 | BABYLON.Quaternion;
+}
+
+interface Point {

```

Comment:
can we reuse Vector2 to avoid recreating multiple times this interface?

Reply:
Sure